samples/nrf9160/nrf_cloud_mqtt_multi_service/child_image/mcuboot.conf

```diff
@@ -0,0 +1,18 @@
+# MCUboot requires a large stack size, otherwise an MPU fault will occur
```

Comment:
This is probably wishful thinking, but is there any way to make these overlays somehow global, perhaps gated by a KConfig?

Reply:
I heard there is a plan for someone to make them treewide soon.